src/Sarif.UnitTests/Visitors/UpdateIndicesVisitorTests.cs

```diff
@@ -41,7 +41,7 @@ public void UpdateIndicesVisitor_FunctionsWithNullMaps()
         {
             var result = _result.DeepClone();
 
-            var visitor = new UpdateIndicesVisitor(null, null);
+            var visitor = new UpdateIndicesVisitor(null, null, null);
```

Comment:
>UpdateIndicesVisitor [](start = 30, length = 20)

Consider: use named parameter syntax so we know what the `null`s are.

Reply:
right, when every argument is null in a test named 'FunctionsWithNullMap', i don't see the need for this readability.  the need for naming comes during actual utilization. 

things that need to be readable need to be readable. otherwise, they should be compact. still don't care enough about this to push back.

---
In reply to: [249155287](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249155287) [](ancestors = 249155287)

Reply:
I took a look at the fix and it actually, to me, compromises the readability of this test. the test is intended to comprehensively validate a call where all args are null. the call without named parameters makes it easiest to chunk the information allowing this condition to be validated. 

---
In reply to: [249187800](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249187800) [](ancestors = 249187800,249155287)